objects/obj_player/Step_0.gml

```diff
@@ -2,187 +2,216 @@
+		current_rotation_speed = rotation_speed;
 	}else {
 		if (current_rotation_speed > 0) {
-			current_rotation_speed -= rotation_delay
+			current_rotation_speed -= rotation_delay;
 		}else if (current_rotation_speed < 0) {
-			current_rotation_speed += rotation_delay
+			current_rotation_speed += rotation_delay;
 		}
 	}
 	
-	angle += current_rotation_speed
+	angle += current_rotation_speed;
 }else if (dead = false) { //use mouse to angle player
 	
 		if invert = false {
 			if (angle <= point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90) {
-				angle += ((point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed
+				angle += ((point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed;
 			}else if (angle >= point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90) {
-				angle += ((point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed
+				angle += ((point_direction(obj_camera.x,y,mouse_x,y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed;
 			}
 		}else{
 			if (angle <= point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90) {
-				angle += ((point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed
+				angle += ((point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed;
 			}else if (angle >= point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90) {
-				angle += ((point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed
+				angle += ((point_direction(obj_camera.x,y,obj_camera.x - (mouse_x-obj_camera.x),y-mouse_sensitivity) - 90)-angle)/mouse_reanglespeed;
 			}
 		}
 			
-		clamp(angle,-anglemax,anglemax) //cant tilt too far
+		clamp(angle,-anglemax,anglemax); //cant tilt too far
 }
 
 
-image_angle = angle
+image_angle = angle;
 
 //falling
 if (!bouncing) {
-	vspeed += grv
+	vspeed += grv;
 }
 
 //horizontal drag
 if hspeed > 0 {
-	motion_add(180,h_grv)
+	motion_add(180,h_grv);
 }else if hspeed < 0 {
-	motion_add(0,h_grv)
+	motion_add(0,h_grv);
 }
 
 //check for collision with ground below
 
 if (place_meeting(x,y+vspeed,obj_ground_oneway) and vspeed > 0 and !place_meeting(x,y-1,obj_ground_oneway)) {
 	while !(place_meeting(x,y+sign(vspeed),obj_ground_oneway)) {
-		y += sign(vspeed)
+		y += sign(vspeed);
 	}
-	bouncing = true
+	bouncing = true;
 	
-	if (object_get_name(instance_place(x,y+vspeed,obj_ground_oneway).object_index) = "obj_floatingenemy") {
-		instance_place(x,y+vspeed,obj_ground_oneway).being_bounced = true
-	}
+	//if (object_get_name(instance_place(x,y+vspeed,obj_ground_oneway).object_index) = "obj_floatingenemy") {
+		//instance_place(x,y+vspeed,obj_ground_oneway).being_bounced = true
+	//}
```

Comment:
It is best to remove commented-out code. Is there still a need for this? if so can you add a comment why?